test/integration/test_kraken_api_order_book_data_source.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+import math
+import time
+from os.path import join, realpath
+import sys; sys.path.insert(0, realpath(join(__file__, "../../../")))
+
+from hummingbot.market.kraken.kraken_api_order_book_data_source import KrakenAPIOrderBookDataSource
+from hummingbot.core.data_type.order_book_tracker_entry import OrderBookTrackerEntry
+import asyncio
+import aiohttp
+import logging
+from typing import (
+    Dict,
+    Optional,
+    List,
+)
+import unittest
+
+
+class KrakenAPIOrderBookDataSourceUnitTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.ev_loop: asyncio.BaseEventLoop = asyncio.get_event_loop()
+        cls.order_book_data_source: KrakenAPIOrderBookDataSource = KrakenAPIOrderBookDataSource()
+
+    def run_async(self, task):
+        return self.ev_loop.run_until_complete(task)
+
+    def test_get_active_exchange_markets(self):
+        market_data: pd.DataFrame = self.run_async(self.order_book_data_source.get_active_exchange_markets())
```

Comment:
I am getting an unresolved reference here. You might have forget to import it. Also it seems like you have some unused imports.

Reply:
should be fixed (?)